src/NeuralNetwork/NeuralNetworkData.js

```diff
@@ -132,200 +140,119 @@ class NeuralNetworkData {
     const normalizedInputs = inputTensor
       .sub(inputMin)
       .div(inputMax.sub(inputMin))
-    
+
     return normalizedInputs.arraySync();
   }
 
+
+
+  /**
+   * Reshape an array back to it
+   * @param {*} arr 
```

Comment:
this jsDoc seems wrong ?

Reply:
Thanks @jonnytest1 for pointing this out! The pull request is massively in flux right now as we figure out the features and how it should work. When we get further along we'll definitely need to do a pass over all the documentation!

Reply:
by the way : not just documentation the typescript linter can analyze jsodc comments and its an acceptable way to do typesafety in js